tls/ownership/ownership.md

```diff
@@ -1409,15 +1945,33 @@
       **Description:** Service CA configmap contains the data for the PEM-encoded CA signing bundle which will be injected to resources annotated with 'service.beta.openshift.io/inject-cabundle=true'
       
 
+      Other locations:
+
+      * file /etc/docker/certs.d/image-registry.openshift-image-registry.svc.cluster.local:5000/ca.crt
+      * file /etc/docker/certs.d/image-registry.openshift-image-registry.svc:5000/ca.crt
+      
+
 2. ns/openshift-kube-controller-manager configmap/service-ca
 
       **Description:** Service CA configmap contains the data for the PEM-encoded CA signing bundle which will be injected to resources annotated with 'service.beta.openshift.io/inject-cabundle=true'
       
 
+      Other locations:
+
+      * file /etc/docker/certs.d/image-registry.openshift-image-registry.svc.cluster.local:5000/ca.crt
+      * file /etc/docker/certs.d/image-registry.openshift-image-registry.svc:5000/ca.crt
+      
+
```

Comment:
I bet we'll end up finding disputes of "who actually owns this ca bundle" because while the content matches, the actual owner seems more likely to be the image registry.  This is likely a common problem and suggests that even though the content is the same, the thing we're actually providing ownership for is the file itself.